addon/intl-computed-property.js

```diff
@@ -0,0 +1,34 @@
+import { get, defineProperty } from '@ember/object';
+import ComputedProperty from '@ember/object/computed';
+import { inject as service } from '@ember/service';
+import { assert } from '@ember/debug';
+
+export default class IntlComputedProperty extends ComputedProperty {
+  constructor(fn, ...dependentKeys) {
```

Comment:
I thought about moving the `...dependenKeys` in front, to mimic the invocation of [`computed`](https://www.emberjs.com/api/ember/3.5/functions/@ember%2Fobject/computed), but this would require some extra argument juggling in here.

I'm fine with it though.